go/test/endtoend/tabletmanager/tablet_health_test.go

```diff
@@ -142,7 +137,7 @@ func TestHealthCheck(t *testing.T) {
 	verifyStreamHealth(t, result)
 
 	// then restart replication, make sure we stay healthy
-	err = clusterInstance.VtctlclientProcess.ExecuteCommand("StopReplication", rTablet.Alias)
+	err = clusterInstance.VtctlclientProcess.ExecuteCommand("StartReplication", rTablet.Alias)
```

Comment:
Interesting. I wonder how the test was passing earlier.

Reply:
It's because tablet is expected to report as healthy if replication is stopped (until lag becomes too high). We could enhance the test to also verify that replication is running. We could do it in a different PR.